Content.Shared/CCVar/CCVars.cs

```diff
@@ -20,9 +20,78 @@ public sealed partial class CCVars : CVars
     public static readonly CVarDef<bool> DebugOptionVisualizerTest =
         CVarDef.Create("debug.option_visualizer_test", false, CVar.CLIENTONLY);
 
-    /// <summary>
-    /// Set to true to disable parallel processing in the pow3r solver.
-    /// </summary>
-    public static readonly CVarDef<bool> DebugPow3rDisableParallel =
-        CVarDef.Create("debug.pow3r_disable_parallel", true, CVar.SERVERONLY);
+        /// <summary>
```

Comment:
This should not be here. Delete all of this. Move the CVars related to Blob to their own folder. No changes should exist in this file.

Reply:
always gotta clean up goobs messes smh

Reply:
you say this but you still did it wrong